charts/incubator/youtrack/values.yaml

```diff
@@ -0,0 +1,35 @@
+image:
+  repository: jetbrains/youtrack
+  tag: 2022.2.55618
+  pullPolicy: IfNotPresent
+persistence:
+  data:
+    enabled: true
+    mountPath: /opt/youtrack/data
+  conf:
+    enabled: true
+    mountPath: /opt/youtrack/conf
+  logs:
+    enabled: true
+    mountPath: /opt/youtrack/logs
+  backups:
+    enabled: true
+    mountPath: /opt/youtrack/backups
+  temp:
+    enabled: true
+    mountPath: /opt/youtrack/temp
+podSecurityContext:
+  runAsGroup: 568
+  runAsUser: 568
+securityContext:
+  readOnlyRootFilesystem: false
+  runAsNonRoot: true
+service:
+  main:
+    ports:
+      main:
+        port: 8080
+        protocol: TCP
+        targetPort: 8080
+portal:
+  enabled: true
```

Comment:
Please update order and spacing according to other apps in stable.
This looks like it's pulled from an incubator app.